tools/elf2hunk.c

```diff
@@ -541,11 +585,13 @@ static int relocate
                 break;
 
             case SHN_COMMON:
-                bug("[ELF2HUNK] SHN_COMMON symbol '%s' unsupported\n", symname);
-                      set_error(EINVAL);
-		      
-                return 0;
+                hunk = bssIndex;
+                value = theBSS->size;
+                theBSS->size += (sym.size + 3) & ~3;
 
+                symptr->shindex = htons(bssIndex);
+                symptr->value = htonl(value);
+            break;
```

Comment:
My changes do not touch any present formatting. The wrong indention of 'break' is ONE formatting issue which is less than COUPLE.

I suggest checking the origin file for formatting issues. If that one is fixed, I can fix the patched version too.